pytest_plugins/fixture_markers.py

```diff
@@ -63,6 +63,15 @@ def pytest_generate_tests(metafunc):
         filtered_versions = set(list_params + match_params)
         # default to all supported versions if no filters were found
         for ver in filtered_versions or settings.supportability.content_hosts.rhel.versions:
+            # Skipping RHEL7 client with RHEL9 and above fips-enabled satellite due to incompatibility
+            if (
+                not (
+                    settings.server.version.rhel_version == '7'
+                    or settings.server.version.rhel_version == '8'
```

Comment:
With current logic for FIPS/Components jobs in CI, it'll be set as either `7.y` and `8.y`, where with `.y` this condition might get false
```
rhelVersion = inventory[0]._broker_args.deploy_rhel_version`
.
.
env.ROBOTTELO_server__version__rhel_version = "@str ${rhelVersion}"
```